main.go

```diff
@@ -540,24 +540,17 @@ func main() {
 		cluster.SslOpts = sslOpts
 	}
 
-	// Use separate session to create the keyspace which we will use in the main session
-	prepare_session, err := cluster.CreateSession()
+	session, err := cluster.CreateSession()
 	if err != nil {
 		log.Fatal(err)
 	}
-	PrepareDatabase(prepare_session, replicationFactor)
-	prepare_session.Close()
+	defer session.Close()
 
+	PrepareDatabase(session, replicationFactor)
 	// NOTE: set the keyspace explicitly to workaround the following bug:
 	//       https://github.com/gocql/gocql/issues/1621
 	cluster.Keyspace = keyspaceName
 
-	session, err := cluster.CreateSession()
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer session.Close()
-
```

Comment:
Please explain in the commit message the reason behind now using only a single session, instead of the previous behavior of using two sessions (was the previous behavior unneccessary?).

Reply:
Done